doc/rst/developer/bestPractices/StandardModuleStyle.rst

```diff
@@ -168,8 +169,10 @@ Handling Failure
 ----------------
 
 In general, when code in a library encounters erroneous behavior, an appropriate
-Error should be thrown.  This will enable users of the library to potentially
-recover from the bad behavior without having to restart their program.
+Error should be thrown (see the :ref:`spec description <Chapter-Error_Handling>`
+for error handling in general and :mod:`Errors` for a list of some potential
+errors to throw).  This will enable users of the library to potentially recover
+from the bad behavior without having to restart their program.
```

Comment:
I think you should just have `Error` be ':mod:\`Errors\`' and remove `and :mod:Errors for a list of potential errors to throw`

Reply:
Oh, so something like this?

> In general, when code in a library encounters erroneous behavior, an appropriate
:mod:`Errors` should be thrown (see the :ref:`spec description <Chapter-Error_Handling>`
for error handling in general)

Reply:
Or maybe :class:`Error` instead?

Reply:
Yes, or even this so that its `Error` and not `Errors`

> In general, when code in a library encounters erroneous behavior, an appropriate
:class:`Error` should be thrown (see the :ref:`spec description <Chapter-Error_Handling>`
for error handling in general)

Reply:
Okay, looks like it needed the path to the class since it's not linking within the file itself, but I think this should be good now!